src/Compute/Compute/ChangeLog.md

```diff
@@ -20,6 +20,7 @@
 
 -->
 ## Upcoming Release
```

Comment:
no changelog needed for doc changes, please remove these section

Reply:
Updated as suggested.